tests/phpunit/api/v3/JobProcessMailingTest.php

```diff
@@ -86,6 +86,7 @@ public function tearDown() {
     //$this->_mut->clearMessages();
     $this->_mut->stop();
     CRM_Utils_Hook::singleton()->reset();
+    $this->quickCleanup(['civicrm_activity_contact', 'civicrm_activity']);
```

Comment:
The rest of the PR looks good just for cleanliness to keep the quickcleanups all together I was thinking should this go in cleanupMailingTest()? - it's called during setup. I tested running moving this into cleanupMailingTest and it worked.

Reply:
I'm OK to fix that but I'd rather see if I can get this merged first if I can

Reply:
Sure. Not a blocker.